spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/health/SpanCollectorMetricsHandler.java

```diff
@@ -0,0 +1,29 @@
+package org.springframework.cloud.sleuth.health;
+
+public interface SpanCollectorMetricsHandler {
```

Comment:
I think this was a C&P? the literal binding is to SpanReporter I think. Maybe we rename things in this file about the act of reporting. Ex. SpanReporterMetrics or similar